annotations/2.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Annotations (milestone)"
-date: 2020-05-31
+title: "Jakarta Annotations"
+date: 2020-07-15
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Annotations defines a collection of annotations representing common semantic concepts that enable a declarative style of programming that applies across a variety of Java technologies.
 
 * [Jakarta Annotations 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.ca/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0-RC1.pdf) (PDF)
-* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0-RC1.html) (HTML)
+* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0.pdf) (PDF)
+* [Jakarta Annotations 2.0 Specification Document](./annotations-spec-2.0.html) (HTML)
 * [Jakarta Annotations 2.0 Javadoc](./apidocs)
-* [Jakarta Annotations 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Annotations 1.3 TCK](https://download.eclipse.org/jakartaee/annotations/2.0/jakarta-annotations-tck-2.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/annotations/2.0/jakarta-annotations-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/annotations/2.0/jakarta-annotations-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0-RC1](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0-RC1/jar)
+  * [jakarta.annotation:jakarta.annotation-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0/jar)
 
 # Compatible Implementations
 
-* [Eclipse GlassFish]()
+* [Eclipse GlassFish](https://eclipse-ee4j.github.io/glassfish/download)
```

Comment:
Does Annotation have it's own standalone CI?  If not, how do we merge this before the final Glassfish is available?

Reply:
It's tricky. Jakarta Annotations doesn't have a reference (compatible) implementation because it's just a set of annotations. Also, there are no standalone TCKs, just a signature test. I put there GlassFish because it was listed as a compatible implementation in version 1.3. Putting GlassFish is fine because it implements an annotation processor for Jakarta Annotations API, but I am not sure that it's a compatible implementation in terms how we understand it.
I am suggesting removing this section and possibly add it later when GF6 is released. Thoughts?

Reply:
Based on what we did with 1.3, the following url was used for the CI:  https://eclipse-ee4j.github.io/common-annotations-api/  Given the unique case for Annotations, I'd be fine with this link for the 2.0 CI as well.

Reply:
Fixed